fzf-skim/.bashrc

```diff
@@ -0,0 +1,42 @@
+# Falcon color themes for fzf and skim
```

Comment:
think these comments are redundant - also put this into its own file at `skim/skim.sh`